pinecone/sparse/splade/splade_vector_generation.ipynb

```diff
@@ -0,0 +1,8784 @@
+{
```

Comment:
<p>"Unlike dense embeddings[,] which can be difficult to interpret[,] sparse embeddings map to tokens for easier interpretability."</p><p> </p><p>"sparse-dense index" -&gt; "sparse-dense vectors" (No special index for sparse-dense)</p><p> </p><p>Omit: ["See the companion guide to learn to skip embedding generation"] (redundant)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>